### DISCUSSION TITLE
How can I limit CPU RAM usage in ComfyUI?

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [Comfy-Org/ComfyUI#4841] [Q&A]
Is there a command-line argument to limit CPU RAM usage, not VRAM? I have ample VRAM, but some nodes only work with CPU RAM, and I only have 50GB. I'm experiencing crashes, particularly when creating videos with features like "vid to vid." The error message is "^C," and ComfyUI crashes due to insufficient RAM. The --gpu-only flag doesn't solve the issue; I need to limit CPU RAM usage or perhaps split cross-attention processing. I'm unsure of the best approach, but it needs to target CPU RAM specifically, as I have plenty of GPU VRAM available.

**Comment by b1k3rdude** [comment]
https://deepwiki.com/hiddenswitch/ComfyUI/9.3-memory-optimization
